draft-nygren-dnsop-svcb-httpssvc.md

```diff
@@ -108,24 +108,23 @@ inadequate when the service operator needs to provide a bound
 collection of consistent configuration parameters through the DNS
 (such as network location, protocol, and keying information).
```

Comment:
Reading through both the Abstract and the Intro, it would be nice to edit these a bit more heavily for length:
- Abstract is quite long, would be nice to be one paragraph (minus the parts that will be removed)
- While the note in the abstract does explain the name, it may be nice to explain a bit. Especially since SVCB isn't really obvious as a name.
- The abstract mentions enhancements to performance and privacy, but these aren't summarized at all in the intro. I'd expect some explanation here to enhance the "why should I care".

Reply:
Done and done.